### PR TITLE
[feature] Service Rich Text

### PR DESCRIPTION
**Pull request by jcmonsa**
Add the service Rich Text

Branch:  
feature/serviceRichText

Test page: 
 http://localhost:8080/fonoaudiologia.html

Captura

### CODE REVIEW

**Review by jrmahechad — APPROVED** [review]

File: src/scss/components/_serviceRichText.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  > p {
+    @include body-m-reg;
+
+    color: var(--grey-2);
+    margin: 0;
+    padding-top: 0.8rem;
+  }
+
```
Comment (Alex-Cortes-87) [inline src/scss/components/_serviceRichText.scss:28]: Please make sure to include ordered lists as well as per the user story requirements.

File: src/scss/components/_serviceRichText.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    margin: 0;
+    padding-top: 0.8rem;
+
+    > li {
+      > p {
+        color: var(--grey-2);
+        @include body-m-reg;
+
+        margin: 0;
+        list-style-type: disc;
```
Comment (Alex-Cortes-87) [inline src/scss/components/_serviceRichText.scss]: Here the list-style-type:disc is not being rendered in the page. This attribute should be part of the` ul `tag as per documentation.

**Review by Alex-Cortes-87 — DISMISSED** [review]

File: src/scss/components/_serviceRichText.scss (hunk 2 above)
Comment (jcmonsa) [reply src/scss/components/_serviceRichText.scss]: Hi Alex u are Right, ill change that, ty! :)

File: src/scss/components/_serviceRichText.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.richtext_title {
+  @include heading-4;
+
+  color: var(--brand-navy);
+  margin: 0;
+  padding-top: 6.4rem;
```
Comment (jsebax) [inline src/scss/components/_serviceRichText.scss]: Add this selector to apply the padding to all the `titles` but the first one
```
&:not(:first-of-type) {
    padding-top: 6.4rem;
 }
```

File: src/scss/components/_serviceRichText.scss (hunk 1 above)
Comment (jsebax) [reply src/scss/components/_serviceRichText.scss:28]: Yeah, please apply the styles to the `ol` tags as well

Comment (jcmonsa) [reply src/scss/components/_serviceRichText.scss:28]: done.

File: src/scss/components/_serviceRichText.scss (hunk 3 above)
Comment (jcmonsa) [reply src/scss/components/_serviceRichText.scss]: ty!. done.

**Review by jsebax — APPROVED** [review]

**Pull request merged**